doc/reference/file_system/index.rst

```diff
@@ -3,13 +3,54 @@
 File Systems
 ############
 
+Zephyr RTOS File system Switch (FS) allows applications to mount multiple
```

Comment:
'File system Switch (FS)': Is that an already existing acronym?
If no, maybe it's not too late to think about it more, since it feels greatly confusing.
When talking about File System, FS sounds like File System. If File system Switch is something different from a File System (as I understand), then why not using FSS to disambiguate?

Reply:
I is really common API for multiple FS, which supports a few instances of different FS-es running simultaneously. so this is common multi FS API.

Reply:
Zephyr RTOS File system Switch (FS) sounds strange, isnt this the virtual file system interface (VFS)?

Reply:
VFS -  It was original name i think.  Virtual File System or Virtual Filesystem Switch.

Reply:
This would be less confusing

Reply:
Ok will fix it.